src/Model/DICOM/DICOMDirectorySearch.py

```diff
@@ -60,73 +62,86 @@ def get_dicom_structure(path, interrupt_flag, progress_callback):
             except (InvalidDicomError, FileNotFoundError, PermissionError):
                 pass
             else:
-                if 'PatientID' in dicom_file:
+                if "PatientID" in dicom_file:
                     patient_id = dicom_file.PatientID
                 else:
                     patient_id = "no_id_" + str(files_with_no_patient_id)
                     files_with_no_patient_id += 1
 
-                if "SOPInstanceUID" in dicom_file \
-                        and "SOPClassUID" in dicom_file \
-                        and "Modality" in dicom_file:
-                    new_image = Image(file_path,
-                                      dicom_file.SOPInstanceUID,
-                                      dicom_file.SOPClassUID,
-                                      dicom_file.Modality)
+                if (
+                    "SOPInstanceUID" in dicom_file
+                    and "SOPClassUID" in dicom_file
+                    and "Modality" in dicom_file
+                ):
+                    new_image = Image(
+                        file_path,
+                        dicom_file.SOPInstanceUID,
+                        dicom_file.SOPClassUID,
+                        dicom_file.Modality,
+                    )
                     if not dicom_structure.has_patient(patient_id):
+                        if "SeriesInstanceUID" not in dicom_file:
```

Comment:
This is a logic change:  Skip files that don't have SeriesInstanceUID, but log which file is missing it.